pkgs/top-level/aliases.nix

```diff
@@ -217,6 +217,7 @@ mapAliases ({
   mlt-qt5 = libsForQt5.mlt;  # added 2015-12-19
   mobile_broadband_provider_info = mobile-broadband-provider-info; # added 2018-02-25
   module_init_tools = kmod; # added 2016-04-22
+  mono-zeroconf = throw "deprecated 2019-09-20: abandoned by upstream."
```

Comment:
```suggestion
  mono-zeroconf = throw "deprecated 2019-09-20: abandoned by upstream.";
```

Reply:
I wish git commit --amend would tell me when I've forgotten to git add the change :/ sorry.